dataactvalidator/validation_handlers/validator.py

```diff
@@ -150,19 +143,23 @@ def validate(record,rules,csvSchema,fileType,interfaces):
                 # If type failed, don't do value checks
                 continue
             #Field must pass all rules
+            # temporary fix for long/short colnames until legacy validation rules are removed:
+            # use long colnames in the record header
+            colMapping = interfaces.validationDb.getShortToLongColname()
+            recordLong = {colMapping.get(key,key):value for key, value in record.items()}
```

Comment:
Should these two lines be run for every field?  Maybe they should happen before the loop starts on line 101?